static/js/prompt.js

```diff
@@ -216,6 +244,104 @@ function rgbToHex(r, g, b) {
     return "#" + componentToHex(r) + componentToHex(g) + componentToHex(b);
 }
 
+var app = new Vue({
```

Comment:
Is this paginated only on the frontend? I think it should also be on the backend, which is a bit more tricky, but might be fine for now